addons/mail/static/tests/chatter_tests.js

```diff
@@ -862,5 +863,82 @@ QUnit.test('followers widget: do not display follower duplications', function (a
     form.destroy();
 });
 
+QUnit.module('ActivityMenu', {
```

Comment:
this actually should be in a different file. This file (chatter_tests) is for testing the chatter, not other widgets.  Please add the test in another file